WLC/image_processing/extended_image.py

```diff
@@ -2,12 +2,27 @@
 
 
 class ExtendedImage:
-    def __init__(self, image, x, y, w, h):
+    def __init__(self, image, x_axis, y_axis, width, height, extended_image=None,
+                 show_pic=False, show_line=False, show_word=False, show_char=False):
         self._image = image
-        self._x = x
-        self._y = y
-        self._w = w
-        self._h = h
+        self._x_axis = x_axis
+        self._y_axis = y_axis
```

Comment:
Just out of curiosity, is there a specific reason why we moved from `_x` to `_x_axis`?

Reply:
single char variables aren't pythonic, this makes pylint happier